extensions/aws/processors/FetchS3Object.h

```diff
@@ -27,6 +27,7 @@
 #include <utility>
 #include <vector>
 
+#include "io/StreamPipe.h"
```

Comment:
I don't see an usage of StreamPipe in this file.

Reply:
OutputStreamCallback is defined in that header.